src/convertToObject.js

```diff
@@ -1,18 +1,24 @@
 'use strict';
 
 /**
- * Implement convertToObject function:
- *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
- * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
- *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const stylesObject = {};
+
+  sourceString
+    .split(';')
+    .forEach((rule) => {
+      const [key, value] = rule.split(':').map((part) => part.trim());
```

Comment:
You could also `trim` the elements like this `stylesObject[key.trim()] = value.trim();`, cause there are always only 2 elements in this array and `map` might be an overkill.